wimm.Secundatives/Extensions/ResultMappingExtensions.cs

```diff
@@ -197,5 +194,24 @@ public static class ResultMappingExtensions
             return val.MapError(func);
         }
 
+        /// <summary>
+        /// Transforms a <see cref="Result{T, TError}"/> into a <typeparamref name="U"/> by applying a function if the mapped
+        /// <see cref="Result{T, TError}"/> containts a value, otherwise by applying a function to the 
```

Comment:
```suggestion
/// <see cref="Result{T, TError}"/> contains
```
I'm sure this was from somewhere else. I feel shame for my typing